src/components/users/users_list.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import UserEssential from './user_essential';
+
+function UsersList({ users }) {
+  return (
+    <div className="row">
+      { users.map(u => <UserEssential user={u} wrapperClass="col-sm-4" />) }
```

Comment:
Here you need to add key to the list which you create with unique value https://ru.reactjs.org/docs/lists-and-keys.html